.github/workflows/main.yml

```diff
@@ -35,6 +35,16 @@ jobs:
         cpanm --installdeps .
         cd -
       shell: bash
+    - name: update imap test suite
+      run: |
+        cd /srv/imaptest.git
+        git fetch
+        git remote -v
+        git status
+        git checkout origin/main
+        git clean -f -x -d
+        cd -
+      shell: bash
```

Comment:
Do you need to run `make` here?  After the `git clean -f -x -d`, all the compiled components will be gone.  The comment in the example cassandane.ini for the `[imaptest]` section says it wants the path to a "built but not installed ImapTest", but this one will just be a bare source tree...

(Did you copy from the JMAP-TestSuite job above?  That one doesn't have any build line because it's perl, there's nothing to compile.)

I don't update ImapTest locally very often myself... I'm still running the nightly release from 2019-05-04. 😅 I suppose I probably should make more effort to run a current one, but I don't remember why I don't.